src/components/Tree/styled/spinner.js

```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components';
+import attachThemeAttrs from '../../../styles/helpers/attachThemeAttrs';
+import Spinner from '../../Spinner';
+
+const BrandSpinner = attachThemeAttrs(styled(Spinner))`
+    color: ${props => props.palette.brand.main};
```

Comment:
You don't need to pass a color for the spinner, it receives the brand color from the variant ;)